changelog/3640.breaking.rst

```diff
@@ -0,0 +1 @@
+`aia_color_table`, `eit_color_table` and `suvi_color_table` now only take `astropy.units` quantities instead of strings.
```

Comment:
```suggestion
`~sunpy.visualization.cm.color_tables.aia_color_table`, `~sunpy.visualization.cm.color_tables.eit_color_table` and `~sunpy.visualization.cm.color_tables.suvi_color_table` now only take `astropy.units` quantities instead of strings.
```